driver/src/main.rs

```diff
@@ -44,7 +44,7 @@ use std::time::Duration;
 use structopt::StructOpt;
 use url::Url;
 
-#[derive(Debug, StructOpt)]
+#[derive(Clone, Debug, StructOpt)]
```

Comment:
nit: I don't think `options.clone()` is being used anywhere so its probably not needed here.

Reply:
```suggestion
#[derive(Debug, StructOpt)]
```